### DISCUSSION TITLE
Windows Key Opening Wrong Menu

### DISCUSSION THREAD

**Discussion by LookingForWayOut** [Open-Shell/Open-Shell-Menu#1552] [Q&A] [answered]
Hello.
I am not sure what I did to cause this. I was fixing a M$ issue with Sticky Keys, to which there are three places I have to have it disabled.
Now for whatever reason, when I press the WIN key on keyboard, it opens the W11 menu, not OpenShell. When I click the Start icon, OpenShell opens as normal.
The setting in OpenShell shows to open OpenShell when pressing the key.
![image](https://github.com/Open-Shell/Open-Shell-Menu/assets/11941595/94d60762-a45e-4b20-afc8-bcb2fe2e6600)
I do not know of a Window$ setting to change what WIN key does that I could have changed when disabling Sticky Keys.
Any thoughts?
Thank you.

**Comment by ge0rdi** [comment]
Please, make sure you are using [latest build](https://github.com/Open-Shell/Open-Shell-Menu/releases) on Windows 11.

**Comment by LookingForWayOut** [comment]
Windows is always updated to public releases. Was just updated last night too.

C:\Users\main>systeminfo | findstr /B /C:"OS Name" /C:"OS Version"
OS Name:                   Microsoft Windows 11 Pro
OS Version:                10.0.22621 N/A Build 22621

**Reply by ge0rdi** [reply]
What I meant is to use latest build of `Open-Shell` on Windows 11.

**Reply by LookingForWayOut** [reply]
v4.4.170.

**Reply by ge0rdi** [reply, marked as answer]
https://github.com/Open-Shell/Open-Shell-Menu/releases

**Comment by LookingForWayOut** [comment]
I installed pre-release 4.4.189. That worked. Not sure what it did but it worked. I would not have thought of touching pre-lease versions before. Worked out in my favor this time.
Thank you, @ge0rdi.
